pages/linux/nordnm.md

```diff
@@ -0,0 +1,35 @@
+# nordnm
+
+> This tool automates the importing and secures the usage of NordVPN OpenVPN configurations through Network Manager.
```

Comment:
This is lifted verbatim from the site. We always try to explain it in simpler terms. Otherwise there would be no difference from the original page, which the users can always look up.

How about - "A tool for managing NordVPN OpenVPN configurations" ?

Note that a tldr page is not meant to be exhaustive. Just enough information in a lucid manner to get a beginner started. For complete information, the man page is always there.